### ISSUE TITLE
nakamoto-wallet: `--addresses` parameter is currently ignored

### ISSUE THREAD

**Issue by theStack** [cloudhead/nakamoto#153]
`nakamoto-wallet` specifies a parameter `--addresses`:

https://github.com/cloudhead/nakamoto/blob/9ea71ed71655e41b8b1ec7db02b4b8dea6f9f1ea/wallet/src/main.rs#L15-L17

But the value of that parameter is never read anywhere. In the start of wallet loop, the addresses are either fetched from the database, or if that is empty, from the hardware device:

https://github.com/cloudhead/nakamoto/blob/9ea71ed71655e41b8b1ec7db02b4b8dea6f9f1ea/wallet/src/wallet.rs#L103-L122

Happy to open a PR if someone can point me into the right direction how that parameter was intended. If it's specified, should only those addresses be watched or do we still want to load the ones from the database?

**Comment by vincenzopalazzo** [comment]
>Happy to open a PR if someone can point me into the right direction how that parameter was intended. If it's specified, should only those addresses be watched or do we still want to load the ones from the database?

Right, so we had to put the address inside the db and then nakamoto will be able to load them. So I think we should insert the address passed by `--addresses` inside the db and then leave the code query the databases?
